aiosmtpd/handlers.py

```diff
@@ -6,7 +6,7 @@
 your own handling of messages.  Implement only the methods you care about.
 """
 
-
+import asyncio
```

Comment:
This will fail `tox -e qa`.  Please rebase/merge master and you'll see the failures.  We have a few custom flake8 rules to enforce import order.  E.g.:

```
aiosmtpd/handlers.py:10:1: B403 Shorter non-from import follows longer
aiosmtpd/tests/test_handlers.py:2:1: B403 Shorter non-from import follows longer
```